From: Tomas Krcka <krckatom@amazon.de>
Subject: mm: be less noisy during memory hotplug
Date: Thu, 23 Mar 2023 17:43:49 +0000

Turn a pr_info() into a pr_debug() to prevent dmesg spamming on systems
where memory hotplug is a frequent operation.

Link: https://lkml.kernel.org/r/20230323174349.35990-1-krckatom@amazon.de
Signed-off-by: Tomas Krcka <krckatom@amazon.de>
Suggested-by: Jan H. Schönherr <jschoenh@amazon.de>
Acked-by: David Hildenbrand <david@redhat.com>
Reviewed-by: Pasha Tatashin <pasha.tatashin@soleen.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---


--- a/mm/mm_init.c~mm-be-less-noisy-during-memory-hotplug
+++ a/mm/mm_init.c
@@ -1086,7 +1086,7 @@ void __ref memmap_init_zone_device(struc
 				     compound_nr_pages(altmap, pfns_per_compound));
 	}
 
-	pr_info("%s initialised %lu pages in %ums\n", __func__,
+	pr_debug("%s initialised %lu pages in %ums\n", __func__,
 		nr_pages, jiffies_to_msecs(jiffies - start));
 }
 #endif
_
